supervision/dataset/formats/pascal_voc.py

```diff
@@ -39,6 +39,16 @@ def object_to_pascal_voc(
     return root
 
 
+# Correction functions for VOC XML format as bounding boxes in VOC XML start at (1,1) not (0,0). Refer:
+# https://github.com/roboflow/supervision/issues/144
+def __increment_bbox_for_pascal_voc(xyxy):
```

Comment:
I don't think we need separate functions for that :) It seems a bit excessive.

Reply:
I'd say my rationale for this is derived from [SRP](https://blog.cleancoder.com/uncle-bob/2014/05/08/SingleReponsibilityPrinciple.html). If tomorrow, we want to change something in sv.Detections that adds another `elseIf` here, calling a function is better than again adding +1. 

Having said that, this is mostly useful when the operation/function is subject to change, which I don't think it is here. I'll make the change :) 

```
    for xyxy, mask, _, class_id, _ in detections:
        name = classes[class_id]
        if mask is not None:
            polygons = approximate_mask_with_polygons(
                mask=mask,
                min_image_area_percentage=min_image_area_percentage,
                max_image_area_percentage=max_image_area_percentage,
                approximation_percentage=approximation_percentage,
            )
            for polygon in polygons:
                xyxy = polygon_to_xyxy(polygon=polygon)
                next_object = object_to_pascal_voc(
                    xyxy=__increment_bbox_for_pascal_voc(xyxy),
                    name=name,
                    polygon=polygon,
                )
                annotation.append(next_object)
        else:
            next_object = object_to_pascal_voc(
                xyxy=__increment_bbox_for_pascal_voc(xyxy), name=name
            )
            annotation.append(next_object)
```